benchmarks/transport_geometry_calculate_distances.py

```diff
@@ -0,0 +1,77 @@
+from asv_runner.benchmarks.mark import parameterize
+
+import tardis.transport.frame_transformations as frame_transformations
+import tardis.transport.geometry.calculate_distances as calculate_distances
+from benchmarks.benchmark_base import BenchmarkBase
+
+
+class BenchmarkTransportGeometryCalculateDistances(BenchmarkBase):
+    """
+    Class to benchmark the calculate distances function.
+    """
+
+    @property
+    def model(self):
+        return 5.2e7
+
+    @parameterize(
+        {
+            "Packet params": [
+                {"mu": 0.3, "r": 7.5e14},
+                {"mu": -0.3, "r": 7.5e13},
+                {"mu": -0.3, "r": 7.5e14},
```

Comment:
We don't really need to parameterize this. These values are not different enough to change the benchmark